### PR TITLE
eskip: add comment test cases for ParsePredicates

### PR DESCRIPTION
**Pull request by AlexanderYastrebov**
Add test cases from #2755 discovered by fuzzing.

### CODE REVIEW

**Comment by sepehrdaddev** [comment]
pls also add `"\x2f\x2f\x00\x00\x00\xe6\xfe\x00\x00\x2f\x00\x00\x00\x00\x00\x00\x00\xe6\xfe\x00\x00\x2f\x00\x00\x00\x00"` it was another payload that also has the same behavior.

File: eskip/eskip_test.go [hunk 1]
```diff
@@ -386,16 +386,22 @@ func TestPredicateParsing(t *testing.T) {
 	}, {
 		title: "star notation",
 		input: `*`,
+	}, {
+		title: "invalid fuzz 1",
+		input: `///`,
```
Comment (AlexanderYastrebov) [inline eskip/eskip_test.go:391]: Well, `//` is an eskip comment

File: eskip/eskip_test.go [hunk 2]
```diff
@@ -386,16 +386,22 @@ func TestPredicateParsing(t *testing.T) {
...
+		fail:  true,
+	}, {
+		title: "invalid fuzz 2",
+		input: "\x2f\x2f\x00\x00\x00\xe6\xfe\x00\x00\x2f\x00\x00\x00\x00\x00\x00\x00\xe6\xfe\x00\x00\x2f\x00\x00\x00\x00",
+		fail:  true,
```
Comment (AlexanderYastrebov) [inline eskip/eskip_test.go:396]: `\x2f\x2f...`  == `//...` - a comment

**Comment by sepehrdaddev** [comment]
:+1:

**Comment by szuecs** [comment]
:+1:

**Comment by AlexanderYastrebov** [comment]
We should also fix `ParseFilters("// a comment")` - currently it fails because internally parser converts it into `* -> // a comment -> <shunt>` which is an invalid route.

**Pull request merged**